### DISCUSSION TITLE
How do I customize Ory Actions webhook result?

### DISCUSSION THREAD

**Discussion by Yukimune** [ory/.github#88] [Q&A]
I'm trying to use Ory Actions Webhooks according to [Trigger custom logic and integrate with external systems with webhooks](https://www.ory.sh/docs/guides/integrate-with-ory-cloud-through-webhooks#creating-actions-on-the-ory-network).

On Flow-interrupting webhooks, I want to return some verification result, but I can't find more payload's specification and examples.
Do you mind to have any docs I can't find yet?

**Comment by vinckr** [comment]
Hello @Yukimune 

See this example: https://github.com/ory/examples/tree/master/ory-actions/vpncheck-py

TLDR: you get a HTTP 200 or HTTP 400 response back. 
Let me know if that helped!
